project-base/storefront/components/Pages/Customer/OrderDetailContent.tsx

```diff
@@ -246,9 +245,7 @@ export const OrderDetailContent: FC<OrderDetailContentProps> = ({ order }) => {
 
                 {!!order.items.length && (
                     <div className="mt-10">
-                        <Heading className="text-center" type="h2">
```

Comment:
I do not think this was semantically incorrect. Why is it not a heading anymore?

Reply:
Semantically yes, placing not. Title "Your purchase" doesn't say anything meaningful to robots. Also, this content doesn't really matter since it is not accessible by robots.